### PR TITLE
LRDOCS-3169 Add Loading Modules with AUI Script Tag

### PR DESCRIPTION
**Pull request by jbalsas**

### CODE REVIEW

File: develop/tutorials/articles/240-javascript-module-loaders/20-loading-modules-with-aui-script-tag.markdown [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+The `aui:script` tag supports the following options:
+
+- **require**: Requires an AMD module that will be loaded with the [Liferay AMD Module Loader](https://github.com/liferay/liferay-amd-loader#amd-module-loader).
+- **use**: Uses an AlloyUI/YUI module that will be loaded via the YUI loader
+- **position**: The position the script tag is put in the page.  Can be `inline` or `auto`.
+- **sandbox**: If set to true, it will wrap the script tag in an anonymous function.  Also, `$` and `_` will be defined for jQuery and underscore.
+
+## Loading ES2015 and Metal.js Modules [](id=loading-es2015-and-metaljs-modules)
+
+You can use `aui:script` to load your ES2015 and Metal.js modules like this:
```
Comment (mwilliams2014) [inline develop/tutorials/articles/240-javascript-module-loaders/20-loading-modules-with-aui-script-tag.markdown]: @jbalsas The example code is missing beneath this sentence. Was this supposed to be removed? thx!

Comment (jbalsas) [inline develop/tutorials/articles/240-javascript-module-loaders/20-loading-modules-with-aui-script-tag.markdown]: Hey @mwilliams2014, this corresponds to the example below. Simply remove the next line so everything makes sense

File: develop/tutorials/articles/240-javascript-module-loaders/20-loading-modules-with-aui-script-tag.markdown [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+You can also load AMD modules by using the `require` attribute, like this:
```
Comment (mwilliams2014) [inline develop/tutorials/articles/240-javascript-module-loaders/20-loading-modules-with-aui-script-tag.markdown]: thx! Just wanted to make sure.

**Comment by mwilliams2014** [comment]
merged locally. thx!

**Comment by mwilliams2014** [comment]
sent https://github.com/sez11a/liferay-docs/pull/1983

**Pull request closed without merging**